cpp/src/slice2cs/Gen.cpp

```diff
@@ -3223,17 +3177,14 @@ Slice::Gen::DispatcherVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
     _out << fixId(name);
 
     _out << sb;
-
-    for (const auto& op : p->operations())
+    for (const auto& op : p->allOperations())
```

Comment:
Now we just call `allOperations` instead of `operations` followed by `writeInheritedOperations`.

A side effect of this is that now, the order that we generate operations in is reversed. It used to be Derived -> Base, but now it's Derived -> Base.